daemon/mgr/container.go

```diff
@@ -941,6 +941,19 @@ func (mgr *ContainerManager) Update(ctx context.Context, name string, config *ty
 		}
 	}()
 
+	if config.Hostname != "" {
```

Comment:
Just asking, why we cannot update hostname for container not set before?

Reply:
This config is updateConfig.If updateConfig.Hostname is nil, it means hostname not need to be updated. @HusterWan